### PR TITLE
[BIOMAGE-2017] V1 cleanup

### PR DESCRIPTION
**Pull request by kafkasl**
# Background
#### Link to issue 
https://biomage.atlassian.net/browse/BIOMAGE-2017

#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [ ] Tested locally with Inframock  (with latest production data downloaded with biomage experiment pull)
- [ ] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/hms-dbmi-cellenics/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/hms-dbmi-cellenics/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/hms-dbmi-cellenics/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: cf/developer-access-group.yaml [hunk 1]
```diff
@@ -240,3 +240,14 @@ Resources:
...
+
+        - PolicyName: !Sub "can-upload-${Environment}-source"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - 's3:PutObject'
+                Resource:
+                  - !Sub "arn:aws:s3:::biomage-originals-${Environment}-${AWS::AccountId}/*"
```
Comment (ivababukova) [inline cf/developer-access-group.yaml:252]: why do we need this additional permission?

File: infra/cf-state-machine-role.yaml [hunk 2]
```diff
@@ -25,24 +25,6 @@ Resources:
...
-          PolicyDocument:
-            Version: "2012-10-17"
-            Statement:
-              - Effect: Allow
-                Action:
-                  - "xray:PutTraceSegments"
-                  - "xray:PutTelemetryRecords"
-                  - "dynamodb:DescribeTable"
-                  - "xray:GetSamplingRules*"
-                  - "xray:GetSamplingTargets"
```
Comment (ivababukova) [inline infra/cf-state-machine-role.yaml]: why do we not need these xray permissions anymore?

Comment (kafkasl) [reply infra/cf-state-machine-role.yaml]: we do need them, I added them back

**Review by aerlaut — APPROVED** [review]
LGTM

**Pull request merged**